tests/unit/calendar/calendarWrapper.mock.js

```diff
@@ -39,6 +40,6 @@ export default class CalendarWrapper extends React.Component {
 }
 
 CalendarWrapper.propTypes = {
-  initialDates: React.PropTypes.arrayOf(String),
-  minDate: React.PropTypes.string,
+  initialDates: PropTypes.arrayOf(String),
```

Comment:
Not part of your changes but this can't be right, _right_?

Reply:
You mean the fact that we have propTypes on a wrapper/mock or the fact that I ordered the propTypes?

Reply:
`arrayOf(String)` should be `arrayOf(PropTypes.string)`. The first one will basically accept an array of truthy values.

Reply:
@reaktivo I think the `arrayOf` receives the Constructor/Type. How would you do with a custom class/Component? `PropTypes.MyComponent`?

Reply:
Ahh screw it, I do think this is supported, but anyway... Changing 🙄

Reply:
Now I got it... It's not a PropType checker... just a constructor ... I thought the arrayOf would do (internally) `value instanceof ConstructorThatWasPassedInTheArrayOf`.